other/relax/relax_driver.jl

```diff
@@ -0,0 +1,21 @@
+using Flux, Statistics
+using Flux.Optimise: update!
+include("relax.jl")
+
+D = 100
+θ = param(zeros(D))
+c = collect(range(0,1,length = D))
+f(x) = sum((x .- c).^2, dims = 1)
+b = BernoulliRelax(f, Chain(Dense(D, 5, relu), Dense(5, 1)))
+
+ϕ = params(b)
+push!(ϕ, θ)
```

Comment:
What is this doing? Is it combining the sets of parameters into one? I don't think this is better.